rerun_py/src/dataframe.rs

```diff
@@ -469,6 +489,23 @@ pub struct PySchema {
 /// [`RecordingView.schema()`][rerun.dataframe.RecordingView.schema].
 #[pymethods]
 impl PySchema {
+    fn __str__(&self) -> String {
+        self.component_columns()
+            .iter()
+            .filter(|field| {
+                !field.entity_path().contains("link") && !field.component().contains("Indicator")
+            })
+            .map(|col| col.__str__())
+            .collect::<Vec<_>>()
+            .join("\n")
+    }
+
+    // Technically this violates python's `__repr__` definition
```

Comment:
I think it would be slightly more valid if we implement the contents of `__str__` in `__repr__` and then call `self.__repr__()` from `__str__`